pkg/controller/controller/group_controller.go

```diff
@@ -301,3 +322,53 @@ func isPodBelongsToGroup(group *crd.Group, pod *api_v1.Pod) bool {
 	}
 	return false
 }
+
+func getPodVolumesMap(pod api_v1.Pod) map[string]*crd.PersistentVolumeClaim {
+	podVolumes := map[string]*crd.PersistentVolumeClaim{}
+	for j := 0; j < len(pod.Spec.Volumes); j++ {
+		vol := pod.Spec.Volumes[j]
+		if vol.PersistentVolumeClaim != nil {
+			claimName := vol.PersistentVolumeClaim.ClaimName
+			podVolumes[claimName] = collectPersistentVolumeClaim(claimName)
+			podVolumes[claimName].BoundTimes = append(podVolumes[claimName].BoundTimes, pod.GetCreationTimestamp())
+		}
+	}
+
+	return podVolumes
+}
+
+func collectPersistentVolumeClaim(claimName string) *crd.PersistentVolumeClaim {
+	pvc, err := ClientSetInstance.CoreV1().PersistentVolumeClaims("").Get(claimName, meta_v1.GetOptions{})
```

Comment:
Pass same namespace as pod. Assumption: Pod can't have pvc from other namespace. Please verify the same.

Reply:
Done